hydradx/model/amm/stableswap_amm.py

```diff
@@ -257,7 +259,50 @@ def execute_withdraw_asset(
     state.shares -= shares_removed
     state.liquidity[tkn_remove] -= quantity
     agent.holdings[tkn_remove] += quantity
-    return state, agent
+    # return state, agent
+
+
+def execute_remove_shares_curve_style(
+        state: StableSwapPoolState,
+        agent: Agent,
+        shares_removed: float,
+        tkn_remove: str,
+        fail_on_overdraw: bool = True
+):
+    # First, need to calculate
+    # * Get current D
+    # * Solve Eqn against y_i for D - _token_amount
+
+    _fee = state.trade_fee  # / 1.5
```

Comment:
What does the "1.5" mean here? I think 0.003 means a 0.3% fee, so 1.5 seems like it's an (invalid) 150% fee?